### PR TITLE
[FLOC-1202] Send SIGTERM to acceptance run before SIGKILL.

### PR DESCRIPTION
**Pull request by tomprince**
https://clusterhq.atlassian.net/browse/FLOC-1202

This is built on #41, since it requires an update to the slave configuration.

This depends on updating the runner to pay attention to the signal.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,8 +3,7 @@ FROM fedora:20
 ADD tomprince-hybridlogic-fedora-20-x86_64.repo /etc/yum.repos.d/
 #RUN yum upgrade -y
 RUN yum install -y python-devel python-pip gcc libffi-devel openssl-devel
-RUN pip install buildbot==0.8.9 txgithub boto service_identity
-
+RUN pip install buildbot==0.8.10 txgithub boto service_identity
```
Comment (wallrj) [inline Dockerfile]: Is this due to "Fixed master-side source steps to respect the specified timeout when removing files" in the buildbot changelog?
 * http://docs.buildbot.net/current/relnotes/index.html

Or some other reason?

File: flocker_bb/builders/flocker_vagrant.py [hunk 2]
```diff
@@ -182,7 +182,10 @@ def buildTutorialBox():
 def run_acceptance_tests(distribution, provider):
     factory = getFlockerFactory()
     factory.addSteps(_flockerTests(
-        kwargs={'trialMode': []},
+        kwargs={
+            'trialMode': [],
+            'sigtermTime': 5*60,
```
Comment (wallrj) [inline flocker_bb/builders/flocker_vagrant.py]: Worth adding a comment explaining why this is here? E.g. Allow 5 minutes for acceptance test machines to be removed.

File: Dockerfile (hunk 1 above)
Comment (wallrj) [inline Dockerfile]: Ah, I found the answer: http://trac.buildbot.net/ticket/751#comment:10

**Comment by wallrj** [comment]
Thanks @tomprince 

This makes sense. But I've only reviewed the code. Haven't tested in anyway.

I suggested that you add a comment or two, but please merge when you're happy with it.

File: Dockerfile (hunk 1 above)
Comment (exarkun) [inline Dockerfile]: The `SIGTERM` change seems to be in 0.8.9 though - http://docs.buildbot.net/current/relnotes/0.8.9.html - what am I missing?

Comment (tomprince) [inline Dockerfile]: It was broken in 0.8.9. The fix is https://github.com/buildbot/buildbot/commit/312f09175ab53aa89f5de60bca837614dcbf5d70 which was in 0.8.10.

**Pull request merged**